source/docs/build-a-standalone-application.md

```diff
@@ -72,11 +72,25 @@ that same file.
 
 ## Database
 
-To use SQLite in your standalone PHAR application, you need to set the following value in your `.env` file:
-```bash
-DB_DATABASE=/absolute/path/to/database.sqlite
+To use SQLite in your standalone PHAR application, you need to tell Laravel Zero where to place the database in a production environment.
+
+Similar to Laravel, this is configured in `config/database.php` under the `connections.sqlite.database` key. By default this is set to `database_path('database.sqlite')` which resolves to `<project>/database/database.sqlite`. Since we can't modify files within the project once the PHAR is built, we need to store this somewhere on the users computer. A good choice for this would be to create a "dot" folder inside your users home folder. For example:
+
+```diff
+// config/database.php
+'connections' => [
+  'sqlite' => [
+      'driver' => 'sqlite',
+      'url' => env('DATABASE_URL'),
+-     'database' => database_path('database.sqlite'),
++     'database' => $_SERVER['HOME'] . DIRECTORY_SEPARATOR . '.your-project-name' . DIRECTORY_SEPARATOR . 'database.sqlite',
```

Comment:
Bear in mind that `HOME` won't be set on Windows (typical Windows 🙄). We can also probably just use `/` instead of the directory seperators (as they work across all systems, they're converted to `\` on Windows).
```suggestion
+     'database' => $_SERVER['HOME'] . '/.your-project-name/database.sqlite',
```

I think this looks good otherwise (although personally I'd probably use a `$HOME/.config/project-name` directory to keep the home directory cleaner, but I think the way it currently is is more common).

To be honest, I feel like it'd probably be a pretty nice thing to have a helper for this in the core. 🤔 Such as:
```php
app()->getHomeConfigDirectory('database.sqlite');
```

Reply:
Ah good to know about the home directory and separators. 

I like that helper a lot! I've been trying to think of a clean way to perform the initial setup, because what I ended up doing was providing an `install` command that was basically just:

```php
public function handle()
{
  File::ensureDirectoryExists(config('database.connections.sqlite.directory'));
  if (! file_exists(config('database.connections.sqlite.database'))) {
    File::put(config('database.connections.sqlite.database'), '');
    $this->callSilent('migrate');
  }
}
```

Another annoying thing is that in all commands that interact with the DB you need to also ensure that this command has been run at least once otherwise it'll throw an error about the database not existing. Have you come into a nice way to have almost like "middleware" for commands?

Reply:
I haven't really found a nice way, other than some sort of trait that you run before the handle method. Can't find the trait though. 🤔